stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -240,7 +240,7 @@ macro time_imports(ex)
     quote
         try
             Base.Threads.atomic_add!(Base.TIMING_IMPORTS, 1)
-            $(esc(ex))
+            @time "Total imports time" $(esc(ex))
```

Comment:
I think "total imports time" gives the impression its a sum of import times
```suggestion
            @time "Total evaluation time" $(esc(ex))
```

Reply:
For instance, it's possible to do
```
@time_imports include("foo.jl")
```